### PR TITLE
Paypal implement

### PR DESCRIPTION
**Pull request by Flou21**

### CODE REVIEW

File: src/utils/Payment/PaymentUtils.tsx [hunk 1]
```diff
@@ -1,25 +1,63 @@
...
-    return defaultPaymentProvider();
+    if (currentProviders.length >= 1) {
+        return currentProviders;
+    }
+    const defaultPaymentProvider = [StripePaymentProvider()];
+    return defaultPaymentProvider;
+}
+
+export function groupProductsByDuration(products: Product[]): Product[][] {
+    products = products.sort((a: Product, b: Product) => {
```
Comment (matthias-luger) [inline src/utils/Payment/PaymentUtils.tsx:32]: shouldnt products.sort((a,b)=>a.premiumDuration-b.premiumDuration) als work?

**Review by matthias-luger — CHANGES_REQUESTED** [review]

File: package.json [hunk 2]
```diff
@@ -29,7 +29,7 @@
         "uuid": "^8.3.1"
     },
     "scripts": {
-        "start": "craco start",
+        "start": "BROWSER=none craco start",
```
[the remark this answers is not in the document]
Comment (Flou21) [reply package.json:32]: nobody wants that

File: src/api/ApiHelper.tsx [hunk 3]
```diff
@@ -659,6 +659,26 @@ function initAPI(): API {
...
+                type: RequestType.PAYPAL_PAYMENT,
+                data: {
+                    orderId,
+                    days
+                },
+                resolve: (response: any) => {
+                    resolve(response);
+                },
+                reject: (error: any) => {
+                    apiErrorHandler(RequestType.RECENT_AUCTIONS, error, {orderId, days});
```
Comment (matthias-luger) [inline src/api/ApiHelper.tsx:675]: use a variable (look at getRecentAuctions)

**Review by matthias-luger — CHANGES_REQUESTED** [review]

**Comment by Flou21** [comment]
still waiting..

**Pull request merged**